addon/services/scheduler.js

```diff
@@ -81,6 +81,10 @@ const Scheduler = Service.extend({
   },
 
   flushQueue(queueName) {
+    if (this.isDestroyed || this.isDestroying) {
```

Comment:
This seems like it may be the result of test leakage. This is a service, and as such is a singleton and shouldn't be destroyed during the regular running of an application. The only time this will manifest itself is in tests, as #17 suggests. 

Again, this won't be an issue when using the new API, so users should be encouraged to migrate to that.